Server/middleware/verifyJWT.js

```
@@ -48,4 +51,69 @@ const verifyJWT = (req, res, next) => {
+    jwt.verify(refreshToken, refreshTokenSecret, (err, decoded) => {
+      if (err) {
+        const errorMessage =
+          err.name === "TokenExpiredError"
+            ? errorMessages.EXPIRED_REFRESH_TOKEN
+            : errorMessages.INVALID_REFRESH_TOKEN;
+        return res.status(401).json({ success: false, msg: errorMessage });
+      }
+
+      // Authenticity of refreshToken is verified, now we can decode jwtToken for payload
+      const jwtSecret = req.settingsService.getSettings().jwtSecret;
+      const decoded = jwt.verify(jwtToken, jwtSecret, { ignoreExpiration: true });
```

Biome

[error] Shouldn't redeclare 'decoded'. Consider to delete it or rename it.

'decoded' is defined here:


(lint/suspicious/noRedeclare)